BotRoyaleUnity/Assets/Scripts/Debug/ExampleRobotBuilder.cs

```diff
@@ -134,13 +178,14 @@ public static List<JSONArray> ExampleRobotsJSON(int numberOfRobots)
                 robotJSON.Add(JSONRobotPart(5, 5, "block"));
 
             }
+            */
 
             sampleRobots.Add(robotJSON);
         }
         return sampleRobots;
     }
 
-    public static JSONObject JSONRobotPart(int x, int y, string type, string direction = "north")
```

Comment:
This change makes the commented out code not work. Please either re-add this default argument or delete the commented out code that will no longer work